numerics_paper/version2/numerical_chj.tex

```diff
@@ -1084,9 +1096,11 @@ \subsection{Flow past a cylinder}\label{sec:bow_shock}
 $[0,40]\times[0,100]$ with a cylinder of radius 20 centered at $(40,50)$.
 Reflecting boundary conditions are imposed at the surface of the cylinder,
 along with outflow conditions along the rest of the right edge of the domain.
-The depth and velocity are set initially and (for all time) at the other
-boundaries to $h_0=1$, $u_0=5$, for a Froude number of 5.
-We use a $160 \times 400$ uniform Cartesian grid.
+{\color{red} The depth and velocity are set initially and (for all time) at the other
+  boundaries to $h_0=1$, $u_0=5$, $v_0=0$ for a Froude number of 5. Therefore, we do not
+  start with a shock; instead, at the beginning of the simulation, the shock forms
+  near the cylinder and travels to its position at steady state.}
```

Comment:
I think it would be clearer to say "The initial condition does not contain a shock, but due to the presence of the cylinder a bow shock rapidly forms and travels to its eventual steady-state position".

Reply:
changed